File: C++/poleNode.cpp
// Stuart Barclay 07/01/2019

#include "poleNode.h"

int poleNode::getValue()
{
    return this->value;
}

poleNode::poleNode()
{
    this->nextNode = nullptr;
    this->value = -1;
}

void poleNode::setValue(int value)
{
    this->value = value;
}

File: Java/src/poleNode.java
public class poleNode {

    private int value;
    private poleNode nextNode;

    public poleNode(){
        System.out.println("New PoleNode Created");
    }

    public void setNextNode(poleNode nextNode){
        this.nextNode = nextNode;
    }

    public poleNode getNextNode(){
        return this.nextNode;
    }

    public void setValue(int value){
        this.value = value;
    }

    public int getValue(){
        return this.value;
    }
}
